test/AutoRest.TestServer.Tests/Mgmt/TestProjects/XmlDeserializationTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Reflection;
+using System.Text.Json;
+using System.Xml.Linq;
+using NUnit.Framework;
+using Azure.Core;
+using XmlDeserialization;
+using XmlDeserialization.Models;
+using SingletonResource;
+
+namespace AutoRest.TestServer.Tests.Mgmt.TestProjects
+{
+    public class XmlDeserializationTests : TestProjectTests
+    {
+        public XmlDeserializationTests()
+            : base("XmlDeserialization") { }
+
+        [Test]
+        public void ValidateMgmtXmlDeserialization()
```

Comment:
This test covers three different cases. It would be easier to understand in the future what case has failed if it is split into three test cases.

Reply:
changed